src/coins.h

```diff
@@ -167,7 +167,7 @@ class CCoins
         nSize += ::GetSerializeSize(VARINT(nCode), nType, nVersion);
         // spentness bitmask
         nSize += nMaskSize;
-        // txouts themselves
+        // txouts themself
```

Comment:
This should be "themselves" since the txouts are plural. But I think just ``// txouts`` would be sufficient.